### DISCUSSION TITLE
关于dnsmasq的DNS重定向打开问题

### DISCUSSION THREAD

**Discussion by nczyw** [immortalwrt/immortalwrt#1421] [Q&A] [answered]
我家因为要看IPTV，新建了一个IPTV接口，用来专门跑IPTV的流量，如果dnsmasq中的DNS重定向打开了，那IPTV的B面认证无法通过，现象上看，IPTV的DNS也被重定向的，我在排除接口中，添加IPTV，或者在作用接口中，取消IPTV，均无法逃过IPTV接口，DNS也被重定向问题，导致无法观看IPTV。
我想请问，我应该如何配置，既打开了DNS重定向，又不影响IPTV观看的方法呢

**Comment by 1715173329** [comment, marked as answer]
自行配置 DNS 劫持规则即可。除非特殊设备，一般情况下无须强制劫持。

**Reply by nczyw** [reply]
好吧，不会设置，我还是关掉算了
